src/markdown-pages/docs/managers/zos-manager.md

```diff
@@ -60,7 +60,52 @@ compileOnly 'dev.galasa:dev.galasa.zos.manager'
 # <a name="configuring"></a>Configuring 
 
 The following properties are used to configure the z/OS Manager.
- 
+
+To provision a CICS Region for your test you need to configure the following properties as a minimum to connect to a CICS region, even if you do not reference a `@ZosImage` in your Galasa test. This is because CICS Regions sit on a z/OS LPAR, and so to provision and connect to a CICS Region in a test, you also need access to the z/OS image that it sits within to make requests on the CICS Region. You might need to configure additional z/OS-related CPS properties, depending on your test.  
+
+
+```
+zos.dse.tag.[tag].imageid=[IMAGEID] OR zos.cluster.[clusterId].images=[IMAGEID]
+zos.image.[IMAGEID].ipv4.hostname=[IP ADDRESS]
+zos.image.[IMAGEID].credentials=[CREDENTIALID]
+```
+
+You also need to configure the following properties for the [CICS TS Manager](cics-ts-manager):
+
+```
+cicsts.provision.type=dse
+cicsts.dse.tag.[TAG].applid=[APPLID]
+```
+
+<details>
+<summary>Hostname of a z/OS system </summary>
+
+| Property: | Hostname of a z/OS system |
+| --------------------------------------- | :------------------------------------- |
+| Name: | zos.image.[IMAGEID].ipv4.hostname |
+| Description: | A physical TCP/IP hostname value for a z/OS system |
+| Required:  | Yes, if connecting to a CICS region |
+| Default value: | None |
+| Valid values: | A valid TCP/IP hostname   |
+| Examples: | <code>zos.image.IMAGEA.ipv4.hostname=dev.galasa.system1</code><br><code>zos.image.SIMBANK.ipv4.hostname=127.0.0.1</code><br><code>zos.image.IMAGEA.ipv4.hostname=winmvs2a.example.com</code><br> |
+
+</details>
+
+<details>
+<summary>Credentials for logging onto a z/OS system </summary>
+
+| Property: | Credentials for logging onto a z/OS system   |
+| --------------------------------------- | :------------------------------------- |
+| Name: | zos.image.[IMAGEID].credentials |
+| Description: |  Credentials for logging onto a z/OS system |
+| Required:  | Yes, if connecting to a CICS region |
```

Comment:
Not necessarily - I needed this property to run the ZosManagerIVT which does not use a CICS Region but instantiates a `@ZosImage`. So I'd replace this with, 'Yes, if connecting to a z/OS image', as that is implied when connecting to a CICS Region, you're also connecting to z/OS. Perhaps we should check the accuracy of this description with someone also.